torch/csrc/distributed/rpc/python_rpc_handler.cpp

```diff
@@ -25,6 +47,8 @@ PythonRpcHandler::PythonRpcHandler() {
   pyLoadReturnValue_ = getFunction(module, "_load_return_value");
   pySerialize_ = getFunction(module, "serialize");
   pyHandleException_ = getFunction(module, "_handle_exception");
+  typeParser_ = std::make_shared<jit::script::ScriptTypeParser>(
```

Comment:
does this require GIL?

Reply:
probably require GIL when get the python_cu via `torch::jit::get_python_cu()`